sync/v1.proto

```diff
@@ -155,6 +161,11 @@ message PreflightResponse {
   //   `none`: The policy will be applied as written
   optional FileAccessAction override_file_access_action = 15;
 
+  // If set, this will put Santa into standalone mode, where a user can approve
+  // binaries from unknown blocks with TouchID. If set to false, Santa will
+  // revert back to normal operation.
+  optional bool standalone_mode = 16;
```

Comment:
I thought we discussed making standalone a new ClientMode?

Reply:
We'd talked about extending the SNTEventStates but not making it a new mode.  RFC6 has the details of what was proposed. 

We'd talked about making this a flag. 

Making standalone mode into a proper mode will take rewiring the PR https://github.com/northpolesec/santa/pull/86

Reply:
@russellhancox I've made the change to an an operating mode.